File: src/main/java/myLang/response/EmptyResponse.java
package myLang.response;

/**
 * Реализация {@link AbstractResponse} для возвращения пустого {@code null} значения.
 * Релизован для избежания ошибок {@link NullPointerException}.
 */
public class EmptyResponse extends AbstractResponse<Void> {
    public EmptyResponse() {
        super(null);
    }

    @Override
    public Void getResponse() {
        return null;
    }
}

File: src/main/kotlin/myLangKotlin/response/EmptyResponse.kt
package myLangKotlin.response

/**
 * Реализация [Response] для возвращения пустого `null` значения.
 * Релизован для избежания ошибок [NullPointerException].
 */
class EmptyResponse(override var response: Void? = null) : Response<Void?>
